src/expandable/options.rs

```diff
@@ -0,0 +1,43 @@
+use yaml_rust::Yaml;
+
+pub fn times(item: &Yaml, with_items_list: Option<&Vec<Yaml>>) -> usize {
```

Comment:
I think you can move this utility function into the `mod` file and leave a single file representation of all expandable items.

Reply:
You are right. This is much better.
Moved it.